src/odemis/driver/simcam.py

```diff
@@ -266,8 +263,8 @@ def _generate(self):
         if self._focus:
             # apply the defocus
             pos = self._focus.position.value['z']
-            dist = abs(pos - self._metadata[model.MD_FAV_POS_ACTIVE]["z"]) * self._blur_factor
-            logging.debug("Focus dist = %g", dist)
+            dist = abs(pos - self._metadata[model.MD_FAV_POS_ACTIVE]["z"]) / (self.depthOfField.value * 10)
```

Comment:
If I remember correctly the blur factor needed to be quite high to give enough blur. Did you verify that the values you are using now give the right blur?

Reply:
Yes, the 3 LENS_*  metadata were adjusted to get almost exactly the same value (to 2 digits).